specs/resources/labels/createLabelReallocation.yml

```diff
@@ -4,7 +4,8 @@ post:
   summary: Create a label reallocation
   description: Reallocates funds between two labels belonging to the same Verified Customer. Moves the specified amount from the source label to the destination label, creating ledger entries for both. The reallocation only succeeds if the source label has sufficient funds.
   operationId: createLabelReallocation
-  x-speakeasy-name-override: createReallocation
+  x-speakeasy-group: labelReallocations
```

Comment:
should this one be `customers.labelReallocation?`

Reply:
@spencerhunter, let me know what you think!